src/components/payouts/payout-details/index.tsx

```diff
@@ -75,88 +64,95 @@ export const PayoutDetails: React.FC<{ memberId: string }> = ({ memberId }) => {
+          options={entryTypeOptions}
+          name="category"
+          defaultValue=""
+          rules={{
+            required: 'Category is required',
+          }}
+        />
+        <FormInput
+          label="Payout amount"
+          name="amount"
+          defaultValue=""
+          type="number"
+          affix={{
+            content: preferredCurrency,
+            basic: true,
+          }}
+          affixPosition="right"
+          rules={{
+            required: 'Amount is required',
+            pattern: {
+              value: /[^0]/,
+              message: 'Amount cannot be zero',
             },
-          }),
-          {
-            loading: 'Creating payout...',
-            success: () => {
-              resetForm()
-              setConfirmed(false)
-              return 'Payout created'
+          }}
+        />
+        <FormInput
+          label="Reference Id"
+          name="referenceId"
+          defaultValue=""
+          rules={{
+            required: 'Reference Id is required',
+            maxLength: {
+              value: 50,
+              message: 'The reference is too long (max 50 characters)',
             },
-            error: 'Could not create payout',
-          },
-        )
-      }}
-      validationSchema={getPayoutValidationSchema()}
-    >
-      {({ isValid }) => (
-        <VerticalForm onChange={() => setConfirmed(false)}>
-          <Field component={FieldSelect} name="category">
-            <MuiMenuItem value="MARKETING">Marketing</MuiMenuItem>
-            <MuiMenuItem value="REFERRAL">Referral</MuiMenuItem>
-            <MuiMenuItem value="REFUND">Refund</MuiMenuItem>
-          </Field>
-          <Field component={TextField} label="Payout amount" name="amount" />
-          <Field
-            component={TextField}
-            label="Reference Id"
-            name="referenceId"
-          />
+          }}
+        />
+        <FormInput label="Note" name="note" defaultValue="" />
 
-          <Field component={TextField} label="Note" name="note" />
-          {!confirmed ? (
-            <SubmitButton
-              type="button"
-              variant="contained"
-              color="secondary"
-              disabled={!isValid}
-              onClick={(e) => {
-                e.preventDefault()
-                setConfirmed((currentConfirmed) => !currentConfirmed)
-              }}
-            >
-              Confirm payout
-            </SubmitButton>
-          ) : (
-            <SubmitButton
-              type="submit"
-              variant="contained"
-              color="primary"
-              disabled={!isValid}
-            >
-              Create payout
-            </SubmitButton>
-          )}
-        </VerticalForm>
-      )}
-    </Formik>
+        {!confirmed ? (
+          <SubmitButton variation="secondary">Confirm payout</SubmitButton>
+        ) : (
+          <SubmitButton variation="primary">Create payout</SubmitButton>
+        )}
```

Comment:
With using `window.confirm` above
```suggestion
  <SubmitButton variation="primary">Create payout</SubmitButton>
```